src/docker-in-docker/install.sh

```diff
@@ -377,16 +379,24 @@ dockerd_start="$(cat << 'INNEREOF'
         echo "Not setting dockerd DNS manually."
         CUSTOMDNS=""
     fi
+
     set -e
 
+    if [ -z "$DOCKER_DEFAULT_ADDRESS_POOL" ] || [ "$DOCKER_DEFAULT_ADDRESS_POOL" = "" ]
```

Comment:
These two checks are redundant.

```suggestion
    if [ -z "$DOCKER_DEFAULT_ADDRESS_POOL" ]
```

Reply:
```suggestion
    if [ -z "${DOCKER_DEFAULT_ADDRESS_POOL}" ]
```